app/models/concerns/smooch_messages.rb

```diff
@@ -364,6 +409,8 @@ def save_message(message_json, app_id, author = nil, request_type = 'default_req
     end
 
     def smooch_save_tipline_request(message, associated, app_id, author, request_type, associated_obj)
+      message['text'] = message['request_body'] unless message['request_body'].blank?
+      message.delete('request_body')
```

Comment:
Why we need this, Sawy?

Reply:
I define a new key ` request_body`    to set all text the should attach to all items as by using the same `text` key I found two issues :
* Attach `media_url` to long text the item parsed as web-link due to existing URL and lead us to invalid item
* having multiple long text leads us to have multiple items created with same text.

Reply:
Ah OK, thanks Sawy!